src/Telemetry/MaestroApiClient/MaestroApiClientFactory.cs

```diff
@@ -0,0 +1,18 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.DotNet.Maestro.Client;
+
+namespace Microsoft.DotNet.Internal.Maestro
+{
+    public sealed class MaestroApiClientFactory : IMaestroApiClientFactory
```

Comment:
This class doesn't seem to be doing much... It's just a single method call.  What purpose is it serving?  Why doesn't the caller just call "ApiFactory.GetAuthenticated" themselves?

Reply:
I don't remember. Will look.

Reply:
I think I did this because I couldn't mock out ApiFactory for the tests

Reply:
Al, that actually makes sense.  Feels like the parameters could be in an options thing to avoid the caller having to deal with all that?